Tasks/VsTest/constants.ts

```diff
@@ -0,0 +1,15 @@
+export module errorCodes {
```

Comment:
Should start with "Capital letter". Please install tslint plugin in vscode and it should warn these

Reply:
tslint didnt make that suggestion

---
In reply to: [136024181](https://github.com/Microsoft/vsts-tasks/pull/5156#discussion_r136024181) [](ancestors = 136024181)